### ISSUE TITLE
should preserve cgi-style options when setting start_inline option for PHP test failure on 1.5.3

### ISSUE THREAD

**Issue by snecklifter** [asciidoctor/asciidoctor-pdf#1756]
I've started seeing the following failure when building for Fedora 33.

```
Failures:

  1) Asciidoctor::PDF::Converter - Source Rouge should preserve cgi-style options when setting start_inline option for PHP
     Failure/Error: (expect funcname_text[:font_color]).to eql '333333'

       expected: "333333"
            got: "0066BB"

       (compared using eql?)
     # ./spec/source_spec.rb:69:in `block (3 levels) in <top (required)>'

```

I've attempted to backport some patches but it seems the test has undergone significant changes since 1.5.3 was released so was hoping someone could shed some light on this.

Build log is here:

https://kojipkgs.fedoraproject.org//work/tasks/9716/49609716/build.log

Not sure if its relevant but both updated prawn and ttfunk packages were updated around the same time.

**Comment by mojavelinux** [comment]
This is actually related to Rouge, not Prawn or ttfunk. Rouge made a major change to the PHP syntax highlighting (basically by rewriting the whole parser), which resulted in different colors in the output. I fixed this in master, but not in the 1.5.x branch.

**Comment by mojavelinux** [comment]
Resolved by 66b335b8c8709ca8919608619a5ff72128dd336d.
